plugins/pulp_docker/plugins/token_util.py

```diff
@@ -82,12 +82,12 @@ def parse_401_response_headers(response_headers):
     """
     auth_header = response_headers.get('www-authenticate')
     if auth_header is None:
-        raise IOError("401 responses are expected to conatin authentication information")
+        raise IOError("401 responses are expected to contain authentication information")
     auth_header = auth_header[len("Bearer "):]
 
     # The remaining string consists of comma seperated key=value pairs
     auth_dict = {}
-    for key, value in (item.split('=') for item in auth_header.split(',')):
+    for key, value in (item.split('=') for item in auth_header.split('",')):
```

Comment:
I think the ``.split('",')`` may be insufficient. The value could be a number instead of a quoted string, as see in the example in the RFC: https://tools.ietf.org/html/rfc7235#section-4.1

See ``type=1`` in that example.

Since the rules for parsing the header value seem complicated, is there something available that could do it for us? Does ``requests`` have something to help with this? Or python standard library? Or something else?